language-plutus-core/generators/Language/PlutusCore/Generators/Interesting.hs

```diff
@@ -224,7 +225,7 @@ genIfIntegers = do
     size <- genSizeDef
     let typedIntS = TypedBuiltinSized (SizeValue size) TypedBuiltinSizedInt
         intS = typedBuiltinToType typedIntS
-    TermOf b bv <- genTermLoose TypedBuiltinBool
+    TermOf b bv <- genTermLoose $ TypedBuiltinDyn @Bool
```

Comment:
What's wrong with `TypedBuiltinBool`?

Reply:
`TypedBuiltinBool` was a useless and restrictive corner case that I removed. It wouldn't allow to typify `verifySignature`.